index.html

```diff
@@ -1,27 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Weather app</title>
-        <!-- importing stylesheet -->
-        <link rel="stylesheet" href="styles.css" />
-        <!-- importing javascript file -->
-        <!-- defer means that the javascript file will be executed after 
-        everything else is loaded -->
-        <script src="app.js" defer></script>
-    </head>
-    <body>
-        <main>
-            <header>
-                <h1>The Weather app</h1>
-                <p id="date-display"></p>
-            </header>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather app</title>
+    <!-- importing fonts -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kalam&family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+    <!-- importing bootstrap -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles.css" />
 
-            <!-- displays the location in latitude and longitude -->
-            <p id="location">the position should be displayed here!</p>
-            <p id="temperature">the temperature should be displayed here!</p>
-            <footer></footer>
-        </main>
-    </body>
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <main>
+      <div class="card bg-light">
+        <div class="card-header">
+          <h1 class="card-title">
+            <span id="city">Munich</span>, <span id="country">Germany</span>
+          </h1>
+          <span id="date-display"></span>
+        </div>
+
+        <div class="card-body text-center">
+          <div class="row">
+            <div class="col-1"></div>
+            <div class="col-7">
+              <div class="mb-3 mt-3">
+                <h2>
+                  <span id="temperature"> 12°C </span>
```

Comment:
hm...I'll ether leave it empty, because it your application will break in between some loading process user will be able to see 12, and probably won't recognise that data was not loaded correctly